exchange/bitswap/wantmanager.go

```diff
@@ -174,7 +174,7 @@ func (mq *msgQueue) doWork(ctx context.Context) {
 	// send wantlist updates
 	err = mq.network.SendMessage(sendctx, mq.p, wlm)
 	if err != nil {
-		log.Noticef("bitswap send error: %s", err)
+		log.Warningf("bitswap send error: %s", err)
```

Comment:
this should probably just be `log.Infof` -- warnings are things the user may need to be conscious about